docs/api/web-contents.md

```diff
@@ -1302,11 +1302,11 @@ Ignore application menu shortcuts while this web contents is focused.
       be set. If no post data is to be sent, the value will be `null`. Only defined
       when the window is being created by a form that set `target=_blank`.
 
-  Returns `{action: 'deny'} | {action: 'allow', outlivesOpener?: boolean, overrideBrowserWindowOptions?: BrowserWindowConstructorOptions}` - `deny` cancels the creation of the new
-  window. `allow` will allow the new window to be created. Specifying `overrideBrowserWindowOptions` allows customization of the created window.
+  Returns `WindowOpenHandlerResponse` -  When set to `{ action: 'deny' }` cancels the creation of the new
+  window. `{ action: 'allow' }` will allow the new window to be created. Specifying `overrideBrowserWindowOptions` allows customization of the created window.
   By default, child windows are closed when their opener is closed. This can be
-  changed by specifying `outlivesOpener: true`, in which case the opened window
-  will not be closed when its opener is closed.
+  changed by specifying `outlivesOpener: true`, in which case the opened window will not be closed when its opener is closed.
+  Specifying `createWindow` allows to customize the way that window is created.
   Returning an unrecognized value such as a null, undefined, or an object
   without a recognized 'action' value will result in a console error and have
   the same effect as returning `{action: 'deny'}`.
```

Comment:
This paragraph is more or less repeating the docs in WindowOpenHandlerResponse, we should probably remove this and fold it into window-open-handler-response.md.